.github/workflows/linter.yml

```diff
@@ -46,12 +46,11 @@ jobs:
       # Run Linter against code base #
       ################################
       - name: Lint Code Base
-        uses: github/super-linter@v3.15.1
```

Comment:
I think it it possible to do something similar to `@v3.*` here? I don't fear any trouble from always loading the last stable version.